### DISCUSSION TITLE
Content Fragment Image

### DISCUSSION THREAD

**Discussion by DigiLiviu** [Adobe-Marketing-Cloud/asset-share-commons#999] [Q&A]
Hi everyone,

How can I use the image component with a Content Fragment? As far as I can see, the image component uses an asset's renditions to render an image and I don't believe Content Fragments have renditions? Maybe I'm not looking in the right place. 

In out use case, there is a thumbnail field that is added to our CF model. We'd like that thumbnail field to be used as the thumbnail for the CF in search results. Is there a way to do that without modifying the Image Model java file?

Thanks,

Liviu

**Comment by davidjgonzalez** [comment]
You cannot use the image core
Component to do this.

the most straightforward way would be to make a preview CF component has has logic to look up the image URL in its sling model. 

Trying to delegate to the image component is going to be tricky and won't get you much - since in asset shares, you don't need to worry as much about Adaptive imaging.
